specs/lib/updateVersionSpec.js

```diff
@@ -0,0 +1,265 @@
+                        technique: 'technique',
+                        values: {
+                            ambient: [0.0, 0.0, 0.0, 1.0]
+                        }
+                    }
+                }
+            },
+            meshes: {
+                mesh: {
+                    primitives: [
+                        {
+                            primitive: WebGLConstants.TRIANGLES
+                        }
+                    ]
+                }
+            },
+            nodes: {
+                node: {
+                    rotation: [0.0, 0.0, 1.0, 0.0],
+                    instanceSkin: {
+                        skeletons: ['skeleton'],
+                        skin: 'skin',
+                        meshes: ['mesh']
+                    }
+                }
+            },
+            techniques: {
+                technique: {
+                    pass: 'pass',
+                    passes: {
+                        pass: {
+                            instanceProgram: {
+                                attributes: {
+                                    attribute: 'TEST_ATTRIBUTE'
+                                },
+                                program: 'program',
+                                uniforms: {
+                                    uniform: 'TEST_UNIFORM'
+                                }
+                            },
+                            states: ['TEST_STATE']
+                        }
+                    }
+                }
+            }
+        };
+        updateVersion(gltf, {
+            targetVersion: '1.0'
+        });
+        expect(gltf.asset.version).toBe('1.0');
+        expect(gltf.asset.profile).toEqual({});
+        expect(gltf.version).not.toBeDefined();
+        expect(gltf.extensionsUsed).toEqual(['extension']);
+        var material = gltf.materials.material;
+        expect(material.technique).toBe('technique');
+        expect(material.values).toEqual({
+            'ambient': [0.0, 0.0, 0.0, 1.0]
+        });
+        var primitive = gltf.meshes.mesh.primitives[0];
+        expect(primitive.primitive).not.toBeDefined();
+        expect(primitive.mode).toBe(WebGLConstants.TRIANGLES);
+        var node = gltf.nodes.node;
+        expect(node.rotation).toEqual([0.0, 0.0, 0.0, 1.0]);
+        expect(node.skeletons).toEqual(['skeleton']);
+        expect(node.skin).toBe('skin');
+        expect(node.meshes).toEqual(['mesh']);
+        var technique = gltf.techniques.technique;
+        expect(technique.pass).not.toBeDefined();
+        expect(technique.passes).not.toBeDefined();
+        expect(technique.attributes).toEqual({
+            attribute: 'TEST_ATTRIBUTE'
+        });
+        expect(technique.program).toBe('program');
+        expect(technique.uniforms).toEqual({
+            uniform: 'TEST_UNIFORM'
+        });
+        expect(technique.states).toEqual(['TEST_STATE']);
+    });
+
+    it('updates glTF from 1.0 to 1.1', function() {
```

Comment:
> If it's undefined, we could assume 1.0. I'm just concerned that this could cause some strange behavior down the line somewhere.

Probably OK.  In practice, I suspect virtually 0% of users use 0.8, and post 1.0, `asset` is required.